### ISSUE TITLE
Please help me understand how to use and modify these files

### ISSUE THREAD

**Issue by ElectricRCAircraftGuy** [themachinist/gcode-syntax-highlighting#5]
Please teach me and give me some quick pointers to be able to modify your package myself. 

It's a very small package, with just 2 files, but I have some questions:

1. Why 2 duplicate files in different formats? `GcodeSyntaxHighlightingDef.tmLanguage` is XML and `GcodeSyntaxHighlightingDef.YAML-tmLanguage` is YAML, but they contain the same data! Why? Can I delete one?
2. Where are the colors actually set? How do you choose names and things?
3. How do you package it up and put it in the package control manager, [here](https://packagecontrol.io/packages/G-code%20Syntax%20Highlighting)?

I'd really appreciate some help on this and I can modify some things in the package myself.

Note to self: **Useful links so far:**

1. https://packagecontrol.io/docs/customizing_packages
2. https://packagecontrol.io/packages/G-code%20Syntax%20Highlighting

**Comment by themachinist** [comment]
If you want to contribute to this repository then you can use Git to clone this repository, make your changes, and submit a pull request. After that I would review the changes and merge them into the main branch, if approved. There may be some intermediary steps I'm forgetting as it's been 6 - 7 years since I've done any coding.

**Comment by ElectricRCAircraftGuy** [comment]
Hi @themachinist , I've started a new stand-alone project here: https://github.com/ElectricRCAircraftGuy/sublime_gcode.

I plan on keeping it maintained. If it would help you so you don't have to maintain yours anymore, feel free to update your readme to redirect people to my repo. I'll see if I can get it into the Package Control system too.

My version isn't complete, but I have it handling comments and a lot of the syntax already. 

I posted this comment here too: https://github.com/themachinist/gcode-syntax-highlighting/issues/4#issuecomment-1073553885
